### PR TITLE
feat: [CI-8134]: removed set -x from run command

### PR DESCRIPTION
**Pull request by devkimittal**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/50560)

### CODE REVIEW

**Comment by devkimittal** [comment]
k8s execution
Screenshot 2023-07-14 at 2 35 08 PM


vm execution
Screenshot 2023-07-14 at 2 28 39 PM

**Comment by devkimittal** [comment]
trigger uts

**Comment by devkimittal** [comment]
tested for windows as well.

**Comment by devkimittal** [comment]
trigger uts

**Comment by devkimittal** [comment]
trigger uts

**Comment by devkimittal** [comment]
trigger uts

File: 332-ci-manager/service/src/main/java/io/harness/ci/execution/serializer/SerializerUtils.java [hunk 1]
```diff
@@ -165,6 +169,20 @@ public static String getEarlyExitCommand(ParameterField<CIShellType> parametrize
     return cmd;
   }
 
+  public static String appendPrintCommand(String command) {
```
Comment (raghavharness) [inline 332-ci-manager/service/src/main/java/io/harness/ci/execution/serializer/SerializerUtils.java:172]: `prependPrintCommand` makes more sense

File: 332-ci-manager/service/src/main/java/io/harness/ci/execution/serializer/SerializerUtils.java [hunk 2]
```diff
@@ -165,6 +169,20 @@ public static String getEarlyExitCommand(ParameterField<CIShellType> parametrize
...
+    String lines[] = command.split("\n");
+    String boldYellowColor = "\u001b[33;1m";
+    StringBuffer printCommand =
+        new StringBuffer(format("echo \"%sExecuting the following command(s):", boldYellowColor));
+    for (String line : lines) {
+      printCommand.append(format("\n%s" + line, boldYellowColor));
+    }
```
Comment (raghavharness) [inline 332-ci-manager/service/src/main/java/io/harness/ci/execution/serializer/SerializerUtils.java:179]: Do we need to do this ? Can we set it similar to this ?
https://github.com/harness/harness-core/blob/66192457ce3cbae00683e75e34a9c8463a560a8c/product/ci/addon/tasks/run.go#L155

**Review by raghavharness — CHANGES_REQUESTED** [review]

**Comment by devkimittal** [comment]
trigger uts

File: 332-ci-manager/service/src/main/java/io/harness/ci/execution/serializer/SerializerUtils.java (hunk 2 above)
Comment (devkimittal) [reply 332-ci-manager/service/src/main/java/io/harness/ci/execution/serializer/SerializerUtils.java:179]: we have to do for each line in java. I tried doing it in one command but doesn't work when new line is there

**Comment by devkimittal** [comment]
trigger uts

**Comment by devkimittal** [comment]
powershell
Screenshot 2023-07-27 at 12 39 18 PM

pwsh
Screenshot 2023-07-27 at 12 40 41 PM

sh and bash
Screenshot 2023-07-27 at 12 48 20 PM

python
Screenshot 2023-07-27 at 12 50 46 PM

**Review by raghavharness — CHANGES_REQUESTED** [review]

**Comment by devkimittal** [comment]
triger uts

**Comment by devkimittal** [comment]
trigger uts

**Review by raghavharness — APPROVED** [review]

**Pull request merged**